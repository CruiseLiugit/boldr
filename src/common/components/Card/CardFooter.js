/* @flow */
import React from 'react';

type Props = {
  children: ReactElement
};
const CardFooter = (props: Props) => {
  return (
    <div className="boldr-card__footer">
      { props.children }
    </div>
  );
};

export default CardFooter;
