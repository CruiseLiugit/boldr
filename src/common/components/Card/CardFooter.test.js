import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import CardFooter from './CardFooter';

it('<CardFooter />, should render', () => {
  const wrapper = shallow(<CardFooter>Hey</CardFooter>);
  const wrapperInner = wrapper.find('.boldr-card__footer');
  expect(wrapperInner.is('.boldr-card__footer')).toBe(true);
});

it('renders snapshot', () => {
  const wrapper = shallow(<CardFooter>Hey</CardFooter>);
  expect(shallowToJson(wrapper)).toMatchSnapshot();
});
