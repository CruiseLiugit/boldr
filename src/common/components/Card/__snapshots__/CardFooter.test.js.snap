exports[`test renders snapshot 1`] = `
<div
  className="boldr-card__footer">
  Hey
</div>
`;
